code/modules/projectiles/guns/energy/special.dm

```diff
@@ -127,10 +127,10 @@
 
 	heat = 3800
 	usesound = list('sound/items/welder.ogg', 'sound/items/welder2.ogg')
-	tool_behaviour = TOOL_WELDER
-	toolspeed = 0.7 //plasmacutters can be used as welders, and are faster than standard welders
+	tool_behaviour = TOOL_DECONSTRUCT
```

Comment:
should prob communicate with gristlebee about #3145..

Reply:
communicated, going to wait until the PR is in to do discussed tweaks